node/node.py

```diff
@@ -335,15 +341,10 @@ def form_private_key_reveal_transaction(self):
         return tx
 
     def form_penalize_violators_transaction(self, conflicts):
-        # TODO can be deprecated on get block by ancestor complete logic
-        for conflict in conflicts:
-            block = self.dag.blocks_by_hash[conflict]
-            self.network.broadcast_block(self.node_id, block.pack())
         self.logger.info("Forming transaction with conflicting blocks")
-        self.logger.info(conflict.hex())
         node_private = self.block_signer.private_key
         tx = TransactionFactory.create_penalty_transaction(conflicts, node_private)
-        return tx
+        return [tx]
```

Comment:
Why return array here? If you need to return array, why left singular name?

Reply:
corrected